scheduler/src/cook/mesos/api.clj

```diff
@@ -1111,8 +1112,9 @@
         ;; if the query params are coerceable, both :job and "job" keys will be
         ;; present, but in that case we only want to read the coerced version
         query-jobs (or query-jobs-coerced (mapv #(UUID/fromString %) query-jobs-uncoerced))
+        body-job (vectorize (get-in ctx [:request :body-params :job]))
```

Comment:
So, since the param I'm handling here is actually part of the body, I probably shouldn't use vectorize here (we know it is going to be a single value).  I will change it.  Thanks for the review, it got me thinking!